Realm/RealmSwift/List.swift

```diff
@@ -39,30 +56,69 @@ public final class List<T: Object>: ListBase, SequenceType {
 
     // MARK: Initializers
 
+    /// Creates a `List` that holds objects of type `T`.
     public override init() {
         super.init(array: RLMArray(objectClassName: T.className()))
     }
 
+    /**
+    Create a `List` by passing in the `RLMArray` to be wrapped.
+
+    :param: rlmArray `RLMArray`.
+    */
```

Comment:
They make the code far less readable and they're mostly worthless noise when not used for the documentation generator. Good documentation for a user of an API and useful comments for people working on the code have little in common.